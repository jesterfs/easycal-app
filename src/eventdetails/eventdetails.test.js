import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import EventDetails from './eventdetails'
import renderer from 'react-test-renderer';

const match = {
    params: { eventId: '1' }
 };

describe('App component', () => {
    
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <EventDetails 
            match={match}
        />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <EventDetails 
            match={match}
        />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});