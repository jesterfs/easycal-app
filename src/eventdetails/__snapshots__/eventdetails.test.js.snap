// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the UI as expected 1`] = `
<div>
  <h1>
    Not Found
  </h1>
</div>
`;
